### ISSUE TITLE
When adding a new Pet, it doesn't appear in Owner's Pet list

### ISSUE THREAD

**Issue by jnz73** [springframeworkguru/sfg-pet-clinic#95]
In the newly added Pet, the OWNER_ID column in the database is null.

**Comment by cassiusvm** [comment]
Hello,

Please, see: https://github.com/cassiusvm/sfg-pet-clinic/commit/d452aadf8955d790884b0d9567838d0635271b1f

**Comment by jnz73** [comment]
Yes, this way works. Adding hat line to the processUpdateForm method in the PetController solves Issue #99. I was wondering if is possible also to set the owner in the pet object through the thymeleaf form adding an hidden input (the same way we pass the Id) to fix both issues.
